cumulus/parachains/runtimes/assets/asset-hub-rococo/src/lib.rs

```diff
@@ -175,8 +179,36 @@ impl frame_system::Config for Runtime {
 	type SS58Prefix = SS58Prefix;
 	type OnSetCode = cumulus_pallet_parachain_system::ParachainSetCode<Self>;
 	type MaxConsumers = frame_support::traits::ConstU32<16>;
+	type SingleBlockMigrations = SingleBlockMigrations;
+	type MultiBlockMigrator = MultiBlockMigrator;
+}
+
+// We destine 80% of the block's weight to MBMs.
+parameter_types! {
+	pub MbmServiceWeight: Weight = Perbill::from_percent(80) * RuntimeBlockWeights::get().max_block;
+}
+
+impl pallet_migrations::Config for Runtime {
+	type RuntimeEvent = RuntimeEvent;
+	#[cfg(not(feature = "runtime-benchmarks"))]
+	type Migrations = MultiBlockMigrations;
+	#[cfg(feature = "runtime-benchmarks")]
+	type Migrations = pallet_migrations::mock_helpers::MockedMigrations;
+	type CursorMaxLen = ConstU32<1024>;
+	type IdentifierMaxLen = ConstU32<256>;
+	type MigrationStatusHandler = ();
+	type FailedMigrationHandler = FreezeChainOnFailedMigration;
+	type MaxServiceWeight = MbmServiceWeight;
+	type WeightInfo = ();
 }
 
+#[cfg(not(feature = "runtime-benchmarks"))]
+type MultiBlockMigrations = assets_common_migrations::v1::Migration<
```

Comment:
I think migrations dont currently know about pallet versions, so you probably had to write the pallet version manually?  
We can make a wrapper for that.

Reply:
I just chose `v1` as a place to put this first migration. We could have more in the future. It's not tied to the version of any pallet. I could just give it a name instead of using `v1`.